can_tools/scrapers/official/NC/nc_vaccine.py

```diff
@@ -34,10 +36,111 @@ class NCVaccine(TableauDashboard):
         ),
     }
 
+    def fetch(self):
+        """
+        uses the tableauscraper module:
+        https://github.com/bertrandmartel/tableau-scraping/blob/master/README.md
+        """
+        ts = TS()
+        ts.loads("https://public.tableau.com/views/NCDHHS_COVID-19_Dashboard_Vaccinations/Demographics")
+        workbook = ts.getWorkbook()
+
+        ageWeekly = workbook.getWorksheet("Age_Weekly_Statewide")
+        raceWeekly = workbook.getWorksheet("Race Weekly Statewide")
+        genderWeekly = workbook.getWorksheet("Gender_Weekly_statewide")
+        ethnicityWeekly = workbook.getWorksheet("Ethnicity_Weekly_Statewide")
+
+        frames = [ageWeekly.data, raceWeekly.data, genderWeekly.data, ethnicityWeekly.data]
+
+        return frames
+
     def normalize(self, df: pd.DataFrame) -> pd.DataFrame:
-        df = super().normalize(df)
-        df.location_name = df.location_name.str.replace(
-            " County", ""
-        ).str.strip()  # Strip whitespace
-        df.loc[df["location_name"] == "Mcdowell", "location_name"] = "McDowell"
-        return df
+        timeran = dt.datetime.now()
+
+        age_df = df[0]
+
+        age_df.rename(columns = {
```

Comment:
I'd recommend breaking out each demographic into it's own normalize method (eg `_normalize_age`, `_normalize_sex`, etc...), then calling each of them in the main `normalize` method.

Reply:
Sounds good to me, let me know when you have input on the location_id issue and i'll fix all these in one go